sapi/fpm/fpm/fpm_conf.c

```diff
@@ -738,6 +738,28 @@ static int fpm_worker_pool_shared_status_alloc(struct fpm_worker_pool_s *shared_
 	FPM_WPC_STR_CP(config, shared_config, group);
 	FPM_WPC_STR_CP(config, shared_config, pm_status_path);
 
+	struct key_value_s *kv;
+
+	for (kv = shared_config->php_values; kv; kv = kv->next) {
+		if (!strcasecmp(kv->key, "expose_php")) {
```

Comment:
I added the key check initially in case there was a setting I wasn't aware of that could prevent it from showing the status page - since that's not needed, have removed the check.